docs/release-notes.md

```diff
@@ -1,3 +1,11 @@
+## 0.14.1
+
+November 8th, 2020
+
+### Removed
+
+* `UJSONResponse` was removed (this change was intended to be included in 0.14.0).
```

Comment:
```suggestion
* `UJSONResponse` was removed (this change was intended to be included in 0.14.0). (Pull #1047)
```

Reply:
This suggestion works too. I thiunk it would be nice provide as many hints here as possible because I'm sure there are people who will upgrade and be surprised (but at least that surprise should come early...).

@florimondmanca What about something like this:

```markdown
* `UJSONResponse` has been removed and a note added to the responses documentation instructing users how to construct their own custom JSON Response classes (see: https://www.starlette.io/responses/)
```

If I saw a line like this in a git log or CHANGELOG and I had been hit by this and if I were unsure where to go next, I *think* I'd have enough of a trail of bread crumbs to follow?